src/Core/Common/MultThreadOilPainting/Code.cxx

```diff
@@ -0,0 +1,76 @@
+/*=========================================================================
+ *
+ *  Copyright Insight Software Consortium
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0.txt
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ *
+ *=========================================================================*/
+#include "itkImage.h"
+#include "itkImageFileReader.h"
+
+#include "itkOilPaintingImageFilter.h"
```

Comment:
`itkOilPaintingImageFilter.h` and `itkOilPaintingImageFilter.hxx` are in _Developer_ category. This example should be moved there (or those files should be moved here).

Reply:
I am sure there are many problems like this.  The student I have working on this is not an ITK expert.  I'd rather not confound the current "bulk migration" work with tasks that require historical, domain specific knowledge.  We'll get to that later.

In this phase we are tryin to "purge media wiki" and "capture content in sphinx".